recipes/python-slugify/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "python-slugify" %}
+{% set version = "1.2.1" %}
+{% set sha256 = "501182ec738cc8b743ae5c76c183f4427187ef016257f062b3fa594f60916e34" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+#  entry_points:
+#    - slugify = slugify:main
```

Comment:
The build failed with both `slugify.slugify:main` and `slugify:main`.
Any ideas?